### PR TITLE
[DW-544] Adds sortable tables

### PR DESCRIPTION
**Pull request by gazjoy**

### CODE REVIEW

File: cms/src/main/resources/ckeditor/plugins/a11ychecker/myJs.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function () {
+    alert('my js!');
```
Comment (ngta1) [inline cms/src/main/resources/ckeditor/plugins/a11ychecker/myJs.js]: Assume this file is no longer needed?

File: repository-data/local/src/main/resources/hcm-config/main.yaml [hunk 2]
```diff
@@ -2,7 +2,6 @@
 definitions:
   config:
     /hippo:configuration/hippo:modules/autoexport/hippo:moduleconfig:
-      autoexport:enabled: false
```
Comment (ngta1) [inline repository-data/local/src/main/resources/hcm-config/main.yaml]: this file change/file can be excluded from the commit.

File: cms/src/main/resources/ckeditor/plugins/a11ychecker/myJs.js (hunk 1 above)
Comment (gazjoy) [reply cms/src/main/resources/ckeditor/plugins/a11ychecker/myJs.js]: Yep good catch!

File: repository-data/local/src/main/resources/hcm-config/main.yaml (hunk 2 above)
Comment (gazjoy) [reply repository-data/local/src/main/resources/hcm-config/main.yaml]: Thanks @ngta1 will get this removed.

Comment (gazjoy) [reply repository-data/local/src/main/resources/hcm-config/main.yaml]: @ngta1 should this go in a `.gitignore`?

Comment (ngta1) [reply repository-data/local/src/main/resources/hcm-config/main.yaml]: We usually keep the file in as it sets autoexport to false by default.  Occasionally when we add a new document type/test data, we need autoexport to be switched on to spit out the YAML bootstrap files needed.

**Comment by gazjoy** [comment]
@GeoffHayward I added to the source as the others do, I thought it was odd but this was consistent with the other js files. If there is any refactoring to do we should pick it up as a new story. It might be an idea to do some performance profiling on the frontend and measure these things. What do you think?

**Comment by GeoffHayward** [comment]
Ok, excellent point.

**Pull request merged**